components/Dropdown/Dropdown.jsx

```diff
@@ -169,8 +169,8 @@ DropContainer.displayName = 'DropContainer';
 
 const _getValue = item => (item ? item.value || item.label || item : '');
 const _getLabel = item => (item ? item.label || item.value || item : '');
-const _getImage = item => (item ? item.img : null);
-const _getImageAlt = item => (item ? item.alt : null);
+const _getImage = ({ img }) => img;
+const _getImageAlt = ({ alt }) => alt;
```

Comment:
I didn't get this. What has changed here?

Reply:
There was an update to eslint that added new rules that broke the previous implementation in this code snippet